testing/tests/ui/crate_identifier.rs

```diff
@@ -0,0 +1,63 @@
+use askama::Template;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ crate }}")]
+struct Crate;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% if crate == 12 %}{% endif %}")]
+struct Crate2;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% match crate %}{% endmatch %}")]
+struct Crate3;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let crate %}")]
+struct Crate4;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let crate = 12 %}")]
+struct Crate5;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ super }}")]
+struct Super;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% if super == 12 %}{% endif %}")]
+struct Super2;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% match super %}{% endmatch %}")]
+struct Super3;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let super %}")]
+struct Super4;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let super = 12 %}")]
+struct Super5;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ Self }}")]
+struct Self;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% if Self == 12 %}{% endif %}")]
+struct Self2;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% match Self %}{% endmatch %}")]
+struct Self3;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let Self %}")]
+struct Self4;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{% let Self = 12 %}")]
+struct Self5;
```

Comment:
Could you please add tests for `{{ self.X }}` and `{{ variable.X }}` for *X* in ["self", "Self", "super", "crate"]?